marketplace/manifest.yaml

```diff
@@ -1,11 +1,11 @@
-applicationName: com.spinkube.spinkube
-publisher: "Spinkube"
+applicationName: dev.spinkube.spinkube
+publisher: "Fermyon"
 description: "SpinKube on Azure Marketplace"
 version: 1.0.0 #Must be in the format of #.#.#
 helmChart: "./charts/spinkube-azure-marketplace"
 clusterArmTemplate: "./mainTemplate.json"
 uiDefinition: "./createUIDefinition.json"
-registryServer: "TODO.azurecr.io"
+registryServer: "spinkubemarketplace.azurecr.io"
```

Comment:
**question:** Is this ACR being created already?

Reply:
Yes, it has been created.  It's also linked to the [appropriate MSFT service principal for deep-copying](https://learn.microsoft.com/en-us/partner-center/marketplace-offers/azure-container-technical-assets-kubernetes?tabs=linux%2Clinux2#grant-access-to-your-azure-container-registry).